panther_hardware_interfaces/include/panther_hardware_interfaces/roboteq_data_converters.hpp

```diff
@@ -0,0 +1,283 @@
+// Copyright 2023 Husarion sp. z o.o.
```

Comment:
What if we moved the method implementations from this header file to the cpp file?

Reply:
I think that shorter, one line implementations should stay, I moved longer ones.